### PR TITLE
Kieker 1956 fix time unit

### PR DESCRIPTION
**Pull request by rju**
# Pull Request

## Contribution
This pull request provides the following contribution to Kieker
- Better handling of Timestamps in different record types.

### CODE REVIEW

File: kieker-common/src-gen/kieker/common/record/flow/AbstractEvent.java [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
 /**
  * @author Jan Waller
- * API compatibility: Kieker 1.15.0
```
Comment (DaGeRe) [inline kieker-common/src-gen/kieker/common/record/flow/AbstractEvent.java:27]: Is it intended to change this? If it is changed, it should be changed to 2.0.0 imho.

Comment (rju) [reply kieker-common/src-gen/kieker/common/record/flow/AbstractEvent.java:27]: Yes it should be upgrade to 2.0.0 I'll regenerate the classes.

**Comment by DaGeRe** [comment]
Thanks for the update, but somehow, during merging, the renamed parameter from tomorrow was changed back, which causes a compilation failure: https://github.com/kieker-monitoring/kieker/actions/runs/4607570640/jobs/8142254657?pr=406 could you fix this?

**Review by DaGeRe — APPROVED** [review]
Thanks for the PR! Looks fine to me and can be merged from my point of view.

**Pull request merged**